update/Pos/Update/Poll/PollState.hs

```diff
@@ -57,7 +58,9 @@ data PollState = PollState
       -- | Update proposals for each application
     , _psActivePropsIdx     :: !(HM.HashMap ApplicationName (HashSet UpId))
       -- | Slotting data for this node
-    , _psSlottingData       :: !SlottingData
+    , _psEpochSlottingData  :: !(HashMap EpochIndex EpochSlottingData)
+      -- | Last updated epoch index
+    , _psEpochLastIndex  :: !EpochIndex
```

Comment:
Well, I payed more attention and I see that in this particular case (`PollState`) storing last epoch explicitly doesn't make sense at all. It's not used `getEpochLastIndex` (in `Pure.hs`) and is not update by `putEpochSlottingData`/`delEpochSlottingData`. So I suppose it's a leftover, isn't it?